lib/senders/index.js

```diff
@@ -165,55 +195,89 @@ module.exports = function (log, config, error, db, translator, sender) {
             })
           })
       },
-      sendPasswordChangedNotification: function (email, opts) {
-        return getSafeMailer(email)
-          .then(function (mailer) {
-            return mailer.passwordChangedEmail({
-              email: email,
-              acceptLanguage: opts.acceptLanguage || defaultLanguage,
-              ip: opts.ip,
-              location: opts.location,
-              uaBrowser: opts.uaBrowser,
-              uaBrowserVersion: opts.uaBrowserVersion,
-              uaOS: opts.uaOS,
-              uaOSVersion: opts.uaOSVersion
-            })
-          })
+      sendPasswordChangedNotification: function (emails, opts) {
+        var mailerPromises = emails.map(function (emailRecord) {
```

Comment:
Does the underlying mailer package have the ability to specify multiple destination addresses in a single call, in the same way you'd put multiple `To:` addresses if sending this email by hand?  If so, it might be both more efficient than doing multiple calls here, and more transparent to the user for them email to show all addresses to which it was delivered.